README.md

```diff
@@ -48,147 +26,49 @@ You can find the deployed project at [Tieme Ndo](https://tieme-ndo.netlify.com/)
 
 In the bid to transform smallholder farmers into semi-commercial farmers and agripreneurs, Tieme Ndo empower farmers to take advantage of business opportunities along the agricultural value chain through SAD (Small Agribusiness Development). The aim of SAD is to train farmers on entrepreneurship and value addition.
 
-### 4️⃣ Key Features
-
-- feature one
-- feature two
-- feature three
-- feature four
-- feature five
```

Comment:
These feature should be filled out based on `Solutions` in our Release Canvas document